rustls/src/msgs/deframer.rs

```diff
@@ -42,62 +40,83 @@ impl Default for MessageDeframer {
     }
 }
 
+thread_local! {
+    static BUF: RefCell<Box<[u8; OpaqueMessage::MAX_WIRE_SIZE]>> =
```

Comment:
Suggest `SHARED_BUF` or add some comment. I read the first time did not realized that it is shard :)

Reply:
Done